lib/src/geojson.dart

```diff
@@ -403,8 +408,8 @@ class GeometryCollection extends Geometry {
   GeometryCollection({this.bbox, this.geometries = const []})
       : super.withType(GeoJSONObjectTypes.geometryCollection);
   @override
-  BBox bbox;
-  List<GeometryType> geometries;
+  BBox? bbox;
+  List<GeometryType>? geometries;
```

Comment:
per https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.8

this should probably just be a non-nullable array. If it has 0 items in it, it's still an array with no entries, rather than null.